superset/db_engine_specs/base.py

```diff
@@ -387,7 +387,7 @@ def df_to_sql(cls, df: pd.DataFrame, **kwargs):  # pylint: disable=invalid-name
 
     @classmethod
     def create_and_fill_table_from_csv(
-        cls, form_data: MultiDict, csv_filename: str, database: Database
+        cls, form_data: MultiDict, csv_filename: str, database
```

Comment:
shouldn't this work when using models.Database?

Reply:
Database and model.Database are failed